environments/kubernetes/manifests/jaeger-deployment.yaml

```diff
@@ -21,14 +21,12 @@ spec:
     spec:
       containers:
       - args:
-        - --badger.directory-key=/var/jaeger/store/keys
-        - --badger.directory-value=/var/jaeger/store/values
-        - --badger.ephemeral=false
         - --collector.queue-size=4000
+        - --memory.max-traces 1000
```

Comment:
In this case, shouldn't the argument be wrapped in quotes? Something to improve in the generator? :)